signalfx/resource_signalfx_table_chart_test.go

```diff
@@ -20,6 +20,13 @@ resource "signalfx_table_chart" "mychartTB" {
 	hide_timestamp = true
 	group_by = ["ClusterName"]
 
+	viz_options {
```

Comment:
I understand that it is missing in some other places, but can we add some assertions to test these new `viz_options`s in `TestAccCreateUpdateTableChart` function?

Reply:
Hi @seonsfx , your comment has been fixed in my new commit - please review.